pkgs/by-name/pd/pd/package.nix

```diff
@@ -0,0 +1,79 @@
+{ rev ? "a0f8550034958d71cb998bfc1d029bb02ff83320"
+, hash ? "sha256-0He3Ryqy4RxYuR2HbwGHJxegfOuKJ91B/JI+z2AK8wk="
+
+, lib
+
+, stdenv_32bit
+, fetchFromGitHub
+, fetchpatch
+
+, pkgsi686Linux # SDL2 and zlib
+
+, pkg-config
+, python3
+}: let
+  shortrev = builtins.substring 0 9 rev;
+in stdenv_32bit.mkDerivation rec {
+  pname = "pd";
+  version = shortrev;
+
+  meta = with lib; {
```

Comment:
meta is usually last in mkDerivation

Reply:
I know, but I didn't see any rules about it, does it matter?

Reply:
> I know, but I didn't see any rules about it, does it matter?

Simply a convention — it helps uniformity in the entire codebase